app/assets/javascripts/tylium/pages/activities.js

```diff
@@ -0,0 +1,62 @@
+document.addEventListener('turbolinks:load', function() {
+  var $activitiesGroupsContainer = $('[data-behavior="activities-groups-container"]');
+  var loading = false;
+  var canLoadMore = true;
+
+  if ($activitiesGroupsContainer.length) {
+    var $viewContent = $('[data-behavior="view-content"]');
+
+    $viewContent.on('scroll', function() {
+      var viewHeight = $viewContent.height();
+      var scrollTop = $viewContent.scrollTop();
+      var scrollHeight = $viewContent[0].scrollHeight;
+
+      // 64px is the sum of margin bottom and padding bottom of the content-container class
+      if (canLoadMore && !loading && viewHeight + scrollTop >= (scrollHeight - 64)) {
+        var page = $activitiesGroupsContainer.data('page') + 1;
+        var url = $activitiesGroupsContainer.data('url');
+        $('[data-behavior="activities-spinner"]').show();
+        loading = true;
+
+        $.ajax({
+          url: url,
+          data: { page: page },
+          success: function(data, status, XHR) {
+            var $activitiesGroups = $(data);
+
+            if ($activitiesGroups.length) {
+              $activitiesGroupsContainer.data('page', page);
+
+              var activityDayValues = $.map($('[data-behavior~=activity-day-value]'), function(element, index) {
+                return $(element).text();
+              })
+
+              $.each($activitiesGroups, function(index, activitiesGroup) {
+                var $activitiesGroup = $(activitiesGroup);
+                var timeElementDatetime = $activitiesGroup.find('[data-behavior~=activity-day-value]').attr('datetime');
+                var timeElementInDOM = $activitiesGroupsContainer.find(`[datetime="${timeElementDatetime}"]`);
```

Comment:
```suggestion
                var $timeElementInDOM = $activitiesGroupsContainer.find(`[datetime="${timeElementDatetime}"]`);
```

since this is a jquery object.

Reply:
The variable is storing the datetime data, not the object itself.

Reply:
Thanks, fixed!